pkg/systemd/overview-cards/motdCard.jsx

```diff
@@ -53,8 +53,8 @@ const MotdEditDialog = ({ text }) => {
                        <Button variant='primary'
                                onClick={() => cockpit.file("/etc/motd", { superuser: "try", err: "message" })
                                        .replace(value)
-                                       .done(Dialogs.close)
-                                       .fail(exc => {
+                                       .then(Dialogs.close)
+                                       .catch(exc => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230722-055749-01048432-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)